src/Forms/RequiredFields.php

```diff
@@ -41,6 +52,22 @@ public function __construct()
         parent::__construct();
     }
 
+    /**
+     * Get whether to allow whitespace only as a valid value for a required field
+     */
+    public function getAllowWhitespaceOnly(): ?bool
+    {
+        return $this->allowWhitespaceOnly;
```

Comment:
```suggestion
        return $this->allowWhitespaceOnly ?? static::config()->get('allow_whitespace_only');
```

Reply:
And before you argue that this is a "getter for the property", it's a getter for whether whitespace is allowed. The property is one party of that, and the config is another part.

Reply:
Updated